gcc/rust/rust-session-manager.cc

```diff
@@ -1177,3 +1150,39 @@ TargetOptions::enable_implicit_feature_reqs (std::string feature)
  *  - code generation
  *  - link */
 } // namespace Rust
+
+#include "selftest.h"
```

Comment:
is it possible this can be in a bla_test.cc file? It might be that we pull out a rust-cfg-parser.cc and have a rust-cfg-parser-test.cc. Is that possible?

Reply:
Sure thing! From what I'm seeing, selftests usually live in the same file as the implementation, but we can definitely move them. Either have a `rust-cfg-parser.cc` which contains the `parse_cfg_option` function and the tests associated, or two different files. Let me know

Reply:
Yay for unit testing!

Reply:
I think in GCC, often the unit tests are in the same file as the implementation they're testing, as otherwise you couldn't easily access `static` functions, for example.